src/recipe/parser/about.rs

```diff
@@ -149,40 +85,96 @@ impl About {
     }
 }
 
-fn parse_license_files(node: &Node, jinja: &Jinja) -> Result<Vec<String>, PartialParsingError> {
-    match node {
-        Node::Scalar(s) => {
-            let script = jinja.render_str(s.as_str()).map_err(|err| {
-                _partialerror!(
-                    *s.span(),
-                    ErrorKind::JinjaRendering(err),
-                    label = "error rendering `script`"
-                )
-            })?;
-            Ok(vec![script])
-        }
-        Node::Sequence(seq) => {
-            let mut scripts = Vec::new();
-            for inner in seq.iter() {
-                match inner {
-                    SequenceNodeInternal::Simple(n) => {
-                        scripts.extend(parse_license_files(n, jinja)?)
-                    }
-                    SequenceNodeInternal::Conditional(if_sel) => {
-                        let if_res = if_sel.process(jinja)?;
-                        if let Some(if_res) = if_res {
-                            scripts.extend(parse_license_files(&if_res, jinja)?)
-                        }
-                    }
+impl TryConvertNode<About> for RenderedNode {
+    fn try_convert(&self, name: &str) -> Result<About, PartialParsingError> {
+        self.as_mapping()
+            .ok_or_else(|| _partialerror!(*self.span(), ErrorKind::ExpectedMapping,))
+            .and_then(|m| m.try_convert(name))
+    }
+}
+
+impl TryConvertNode<About> for RenderedMappingNode {
+    fn try_convert(&self, name: &str) -> Result<About, PartialParsingError> {
+        let mut about = About::default();
+        for (key, value) in self.iter() {
+            match key.as_str() {
+                "homepage" | "home" => about.homepage = Some(value.try_convert("homepage")?),
```

Comment:
Same, if you use `key` here for the names its less likely to make mistakes. And its more correct in the context!